src/Middleware/Diagnostics/src/DeveloperExceptionPage/Views/EndpointModel.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Diagnostics.RazorViews
+{
+    internal class EndpointModel
+    {
+        public string DisplayName { get; set; }
+        public string RouteName { get; set; }
+        public string RoutePattern { get; set; }
+        public int? Order { get; set; }
```

Comment:
This could really just be 0 since that's always the default. endpoints/routes always have an order.

Reply:
You think get rid of order and route name then?

Is there any metadata that you want to add? HttpMethod might be useful